docs/source/troubleshooting.rst

```diff
@@ -15,3 +15,30 @@ data was generated.
 See:
 
   * :data:`~dbldatagen.data_generator.DataGenerator.explain`
+
+You may also configure the data generator to produce more verbose output when building the
+dataspec and the resulting data set.
+
+To do this, set the ``verbose`` option to ``True`` when creating the dataspec. For example:
```

Comment:
It would be nice to add an example of output if it's possible, so people would understand what to look for